package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "frak",
-    "releaseName": "Frak",
+    "releaseName": "frak",
```

Comment:
I disagree with this. It's the release name and should be uppercase

Reply:
I fixed it.